### PR TITLE
N21: Add chip-8-rs

### PR DESCRIPTION
**Pull request by JonathanMurray**
Part of https://github.com/rust-gamedev/rust-gamedev.github.io/issues/589

### CODE REVIEW

File: content/news/021/index.md [hunk 1]
```diff
@@ -523,6 +523,22 @@ Try out the [online demo](https://mvlabat.github.io/bevy_egui_web_showcase/index
...
+a _CHIP-8_ emulator with some basic debugging functionality.
+
+When running a game through the emulator, CHIP-8 instructions are listed
+next to the main display, with the currently executed one highlighted. By
+running at a very low clock-frequency (and pausing/resuming) you can step
+through a program one instruction at a time, to better understand how it
+works (or doesn't work!).
+
+See it in action here: https://youtu.be/nVDJ5PZpPfI?t=72
+
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:541]:
```suggestion

[chip-8-rs]: https://github.com/JonathanMurray/chip-8-rs
```

File: content/news/021/index.md [hunk 2]
```diff
@@ -523,6 +523,22 @@ Try out the [online demo](https://mvlabat.github.io/bevy_egui_web_showcase/index
 [bevy_webgl2]: https://github.com/mrk-its/bevy_webgl2
 [version 0.4]: https://github.com/mvlabat/bevy_egui/blob/main/CHANGELOG.md
 
+### [chip-8-rs]
+
+![Screenshot of Pong with debugger](chip-8-rs.jpg)
+_Debugging Pong_
+
+Chip-8-rs (https://github.com/JonathanMurray/chip-8-rs) by @jonathanmurray is
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:531]:
```suggestion
[Chip-8-rs][chip-8-rs] by @jonathanmurray is
```

**Review by 17cupsofcoffee — APPROVED** [review]
The content looks good, just a few tweaks to make the heading link correctly:

File: content/news/021/index.md [hunk 3]
```diff
@@ -523,6 +523,24 @@ Try out the [online demo](https://mvlabat.github.io/bevy_egui_web_showcase/index
...
+[Chip-8-rs][chip-8-rs] by @jonathanmurray is
+a _CHIP-8_ emulator with some basic debugging functionality.
+
+When running a game through the emulator, CHIP-8 instructions are listed
+next to the main display, with the currently executed one highlighted. By
+running at a very low clock-frequency (and pausing/resuming) you can step
+through a program one instruction at a time, to better understand how it
+works (or doesn't work!).
+
+See it in action here: https://youtu.be/nVDJ5PZpPfI?t=72
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:540]: Oops, the linter doesn't like bare URLs...
```suggestion
See it in action on [YouTube][chip-8-rs-video].
```

File: content/news/021/index.md [hunk 4]
```diff
@@ -523,6 +523,24 @@ Try out the [online demo](https://mvlabat.github.io/bevy_egui_web_showcase/index
...
+
+[chip-8-rs]: https://github.com/JonathanMurray/chip-8-rs
```
Comment (17cupsofcoffee) [inline content/news/021/index.md:542]:
```suggestion
[chip-8-rs]: https://github.com/JonathanMurray/chip-8-rs
[chip-8-rs-video]: https://youtu.be/nVDJ5PZpPfI?t=72
```

**Review by ozkriff — APPROVED** [review]

**Comment by 17cupsofcoffee** [comment]
Thank you for the PR!

**Pull request merged**